schema.js

```diff
@@ -6,7 +6,8 @@ module.exports = Joi.object().keys({
     adminLookup: Joi.object().keys({
       // default maxConcurrentReqs to # of cpus/cores * 10
       maxConcurrentReqs: Joi.number().integer().default(cpus().length*10),
-      enabled: Joi.boolean().default(true)
+      enabled: Joi.boolean().default(true),
+      missingMetafilesAreFatal: Joi.boolean().default(false)
     }).unknown(true),
```

Comment:
line 10 here (`unknown(true)`) means it's ok for me to add this new option to all our config files in Mapzen Search right now, yes? If i don't do it now I'm sure we'll forget :P

Reply:
Yes, you can add whatever you want to `imports.adminLookup`.

Reply:
Great. it's done. hopefully we don't decide to last minute bikeshed the name